afisha-client/src/app/shared/services/auth-interceptor.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable, Injector } from '@angular/core';
+import {HttpInterceptor} from "@angular/common/http";
+import {AuthService} from "./auth.service";
+
+@Injectable()
+export class AuthInterceptorService implements HttpInterceptor{
+
+  constructor(private injector: Injector) { }
+
+  intercept(req, next){
+    const auth = this.injector.get(AuthService);
```

Comment:
Why'd you import AuthService through injector?

Reply:
Because I am injection HttpClient in AuthService, and then inject AuthService in AuthInterceptor, that's caused an error.